package.json

```diff
@@ -14,37 +14,38 @@
     },
     "dependencies": {
         "@cypress/code-coverage": "^3.10.0",
-        "@deriv/deriv-api": "^1.0.9",
+        "@deriv/deriv-api": "^1.0.11",
         "@rollup/plugin-babel": "^5.3.1",
-        "@types/react": "^17.0.33",
-        "@types/react-dom": "^17.0.10",
-        "@typescript-eslint/eslint-plugin": "^5.19.0",
-        "@typescript-eslint/parser": "^5.19.0",
-        "@vitejs/plugin-legacy": "^1.8.2",
-        "@vitejs/plugin-react": "^1.3.2",
+        "@types/react": "^18.0.15",
+        "@types/react-dom": "^18.0.6",
+        "@typescript-eslint/eslint-plugin": "^5.30.6",
+        "@typescript-eslint/parser": "^5.30.6",
+        "@vitejs/plugin-legacy": "^2.0.0",
+        "@vitejs/plugin-react": "^2.0.0",
         "@xstate/react": "^3.0.0",
         "@xstate/test": "^0.5.1",
-        "cypress": "^10.2.0",
+        "cypress": "^10.3.0",
         "dialog-polyfill": "^0.5.6",
-        "eslint": "^8.17.0",
+        "eslint": "^8.20.0",
         "eslint-config-prettier": "^8.5.0",
         "eslint-plugin-import": "^2.26.0",
-        "eslint-plugin-prettier": "^4.0.0",
-        "eslint-plugin-react": "^7.30.0",
-        "prettier": "^2.6.2",
-        "prismjs": "^1.27.0",
-        "react": "^17.0.2",
-        "react-dom": "^17.0.2",
-        "react-hook-form": "^7.29.0",
-        "react-query": "^3.34.16",
+        "eslint-plugin-prettier": "^4.2.1",
+        "eslint-plugin-react": "^7.30.1",
+        "prettier": "^2.7.1",
+        "prismjs": "^1.28.0",
+        "react": "^18.2.0",
```

Comment:
I don't think this is needed, we only need to make sure to fix all the security issues running `npm audit`

Reply:
Agree, also React 18 includes some new behaviours that we will have to consider and test before updating